lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -114,17 +137,19 @@ def self.deliver(*args)
         else
           if ae_result.casecmp('error').zero?
             miq_task&.update_message(MiqTask::MESSAGE_TASK_COMPLETED_UNSUCCESSFULLY)
-            message = "Error delivering #{ManageIQ::Password.sanitize_string(options[:attrs].inspect)} for object [#{object_name}] with state [#{state}] to Automate: #{ws.root['ae_message']}"
+            message = "Error delivering #{attr_string} for object [#{object_name}] with state [#{state}] to Automate: #{ws.root['ae_message']}"
             _log.error(message)
+            RequestLogs.create(:log_message => message, :miq_requests_id => $request_id, :object_id => $object_id, :object_type => $object_type)
```

Comment:
I think we discussed this elsewhere, and I'm fine with this for now, but we might want to create a specialized RequestLogger that writes to the table as well as the log.

Reply:
Working on the logger.